TimeTable/TimeTable/Program.cs

```diff
@@ -0,0 +1,152 @@
+                    break;
+            }
+        }
+
+        static void ShowAllEvents()
+        {
+            Console.WriteLine();
+            Console.WriteLine("Here is a list of saved events: ");
+            Database databaseObject = new Database();
+            
+            // View all saved events
+            string queryAllEvents = "SELECT * FROM myCalendar ORDER BY eventDate ASC";
+            SQLiteCommand selectAllCommand = new SQLiteCommand(queryAllEvents, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultAll = selectAllCommand.ExecuteReader();
+            if (resultAll.HasRows)
+            {
+                while (resultAll.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultAll["eventDate"], resultAll["eventName"]);
+                }
+            }
+            ShowMenu();
+        }
+
+        // Display selected date
+        static void SelectOneDay()
+        {
+            // Ask the user to enter any date
+            string enteredDay;
+            Console.WriteLine("Enter the date in the format YYYY-MM-DD: ");
+            enteredDay = Console.ReadLine();
+
+            ShowEventsByDay(enteredDay);
+            ShowMenu();
+        }
+
+        static void ShowEventsByDay(string enteredDay)
+        {
+            // Search and display the selected date
+            Database databaseObject = new Database();
+
+            string query = $"SELECT * FROM myCalendar WHERE eventDate='{enteredDay}'";
+
+            SQLiteCommand selectOneDayCommand = new SQLiteCommand(query, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultFromOneDay = selectOneDayCommand.ExecuteReader();
+            if (resultFromOneDay.HasRows)
+            {
+                while (resultFromOneDay.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultFromOneDay["eventDate"], resultFromOneDay["eventName"]);
+                }
+            }
+            else
+            {
+                Console.WriteLine();
+                Console.WriteLine("You have no events for this day");
+            }
+            
+         }
+
+        static void AddEvents()
+        {
+            // Add event to The calendar
+            Database databaseObject = new Database();
+
+            string query = "INSERT INTO myCalendar (`eventDate`, `eventName`) VALUES (@eventDate, @eventName)";
+            SQLiteCommand myCommand = new SQLiteCommand(query, databaseObject.myConnection);
+
+            databaseObject.OpenConnection();
+
+            // Ask the user to enter a date 
+            string eventDate;
+            Console.WriteLine("Enter the day in the format YYYY-MM-DD: ");
+            eventDate = Console.ReadLine();
+            myCommand.Parameters.AddWithValue("@eventDate", eventDate);
```

Comment:
Read about try and catch clauses to catch exceptions and not break down application when something goes wrong whie saving data to DB.

Reply:
An important point, I will definitely work on it.